challenges/KICKin-Rollin-Challenge.tex

```diff
@@ -0,0 +1,46 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{KICKin\textquotesingle \& Rollin\textquotesingle Challenge}
+
+The objective of this challenge is for a robot to successfully kick a rolling ball from a ramp into a designated goal area. 
+Each team participating will record three attempts spread accross three days of the competition.
+An attempt consist of three kick attempts. 
+
+\subsection{Field Setup}
+
+The challenge will take place on a standard SPL field with its official marking. 
+The ramp will be positioned perpendicularly adjacent to the halfway line, aligned with a virtual line extending from the extremity of the center circle on the opponent's side.
+The ramp can be positioned at varying distances from the robots to modify the difficulty.
+The participating robot can be positioned at the team’s discretion along the virtual line between the center circle and the penalty mark.
```

Comment:
Which direction will the ball roll relative to the robot. I would have thought that the ball should not be rolling directly at the robot and instead should be rolling such that it will pass in front of the robot. If the robot must walk more than a few cm or rotate more than a few degrees then it would seem that the ball would have to move unrealistically slowly or else the robot would not have enough time to move.